plugins/org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/DefaultUiModule.java

```diff
@@ -428,4 +431,17 @@ public void configureIssueCodesPreferenceInitializer(Binder binder) {
 			.to(ConfigurableIssueCodesPreferenceStoreInitializer.class);
 	}
 	
+	/**
+	 * @since 2.9
+	 */
+	public Class<? extends IWorkspaceConfigProvider> bindWorkspaceConfigProvider() {
+		return EclipseWorkspaceConfigProvider.class;
+	}
+	
+	/**
+	 * @since 2.9
+	 */
+	public org.eclipse.core.resources.IWorkspaceRoot bindIWorkspaceRootToInstance() {
+		return ResourcesPlugin.getWorkspace().getRoot();
```

Comment:
We cannot bind IWorkspaceRoot for all UI languages, e.g. RCP apps don't have that on the CP